### DISCUSSION TITLE
Support PriorityAttribute of MsTest

### DISCUSSION THREAD

**Discussion by livioc** [reqnroll/Reqnroll#41] [Ideas]
The MsTest v2 framework has a [PriorityAttribute ](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.priorityattribute) which isn't supported by reqnroll, I'm going to do a pr for it

**Comment by gasparnagy** [comment]
Thx @livioc. Was that working with SpecFlow?

**Reply by livioc** [reply]
No, I fixed it in my projects with a private nuget package using SpecFlow.CustomPlugin, I saw that specflow rep didn't had any activity in previous year so I have never tried to create a public pr for it.
Anyhow thanks so much for reqnroll :)

**Comment by gasparnagy** [comment]
Implemented by https://github.com/reqnroll/Reqnroll/pull/42
